.eslintrc.json

```diff
@@ -18,7 +18,8 @@
     "globalThis": false
   },
   "rules": {
-    "i18n-text/no-en": "off"
+    "i18n-text/no-en": "off",
+    "no-console": "off"
```

Comment:
I am assuming this is only needed for dev work. Ideally, shouldn't be in the PR.

Reply:
Yes, this needs to be in PR, because Dev env. is also important as we are working on open source code, This part is for linting, as we are using console for logging into our code, thats why it is important to allow console so that it will not create problem to while development.